style.css

```diff
@@ -1503,3 +1507,435 @@
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .recent-works .frame-44 {
+    height: 2500px;
+    width: 100vw;
+  }
+
+  .recent-works .frame-44 .frame-44-1 {
+    display: flex;
+    width: 100vw;
+    justify-content: space-around;
+  }
+
+  .recent-works .frame-44 .frame-44-1 img {
+    width: 684px;
+    height: 448px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .recent-works .frame-44 .frame-44-1 .right-block {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    width: 548px;
+    height: 448px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .recent-works .frame-44 .frame-44-1 .right-block h2 {
+    width: 448px;
+    height: 48px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-top: -5px;
+  }
+
+  .form-intro-container p {
+    width: 455px;
+    height: 256px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #262626;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .recent-works .frame-44 .frame-44-1 .right-block p {
+    width: 448px;
+    height: 140px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .frame-43 {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    height: 900px;
+    width: 85vw;
+    margin-top: -5px;
+  }
+
+  .about-me-section {
+    height: 1370px;
+    width: 100vw;
+    top: 2707px;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 130px 0;
+    gap: 24px;
+    position: absolute;
+    width: 100vw;
+    height: 582px;
+    left: 142px;
+    background-image: url(./imgs/aboutBG.svg);
+    background-position: 84% 30%;
+  }
+
+  .about .title {
+    width: 566px;
+    height: 78px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .about .description {
+    width: 566px;
+    height: 148px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .skill {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 130px 0;
+    gap: 152px;
+    position: absolute;
+    width: 100vw;
+    height: 596px;
+    left: 141px;
+    top: 629px;
+  }
```

Comment:
Kindly ensure that the skills section follows the figma design as closely as possible. Consider how the list items are vertically aligned with the headings. Also ensure that the background color matches that of the design

Your design:
![image](https://user-images.githubusercontent.com/20554583/215264308-2f14b53b-633a-489a-8523-c062d5cb0a53.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/215264324-370d419c-8449-469c-af61-c21cfbecf582.png)

Reply:
Done.